packages/@remirror/extension-callout/src/styles.ts

```diff
@@ -0,0 +1,26 @@
+import { css } from 'linaria';
+
+export const editorStyles = css`
+  div[data-callout-type] {
+    background: #eef6fc;
+    border-left: 2px solid #3298dc;
```

Comment:
Default here is info styling, same as the default type.

Reply:
Although on second thoughts, make this is unwise, as default type is configurable

Reply:
> Although on second thoughts, maybe this is unwise

I think agree with this sentiment. Since there will always be a default it's probably best to leave this part unstyled.